documentation/MIGRATION-guide-for-next-generation-management-libraries.md

```diff
@@ -214,9 +224,22 @@ In next-generation, we keep this feature except
+| stop polling                                                               | N/A                                          | `stopPolling()`                                                                                                                                                                                                 |
+| check if the polling stopped                                               | N/A                                          | `isStopped()`                                                                                                                                                                                                   |
+| cancel the operation                                                       | N/A                                          | `cancelOperation()`                                                                                                                                                                                             |
+| get the current operation state                                            | `getPollState()`                             | `getOperationState()`                                                                                                                                                                                           |
+| access the final result                                                    | `getOperationResponse()`                     | `getResult()`                                                                                                                                                                                                   |
+| serialize the poller state                                                 | N/A                                          | `toString()`                                                                                                                                                                                                    |
+| get the most recent response                                               | `getMostRecentResponse()`                    | can be accessed using the [`onResponse`](https://github.com/Azure/azure-sdk-for-js/blob/35f86a68def5022d4c03a32d4c8c0362a0158772/sdk/core/core-client/src/interfaces.ts#L115) callback in the operation options |
+| get the current operation status                                           | `getOperationStatus()`                       | can be accessed using the [`onResponse`](https://github.com/Azure/azure-sdk-for-js/blob/35f86a68def5022d4c03a32d4c8c0362a0158772/sdk/core/core-client/src/interfaces.ts#L115) callback in the operation options |
```

Comment:
Could we have some simple code snippet on some most common scenarios of how to use the poller before and how can they migrate to the next generation,  like the List Operations.

Reply:
Yeah I thought about that but felt like the table comparison of the method names and their behavior would be better because it is succinct and on point. All what the code snippet will do is just call these methods except for the onProgress scenario and the onResponse ones which are more complicated. I can provide examples for those if you're interested but these are not common scenarios.

Reply:
maybe we don't need to provide an example per table row, just one or two examples that containst the track 1 LRO and equivalent of Track 2?

Reply:
Done.